provider/aws/formation/app.json.tmpl

```diff
@@ -290,7 +291,7 @@
             {{ range .Resources }}
               { "Name": "{{ upper . }}_URL", "Value": { "Fn::GetAtt": [ "Resource{{resource . }}", "Outputs.Url" ] } },
             {{ end }}
-            {{ range $k, $v := $.Env }}
```

Comment:
This is a potentially breaking change. However I observed that the old behavior did not set pre-defined variables in a service, and passed through variables that were not white listed.